lib/managers/files.js

```diff
@@ -4,23 +4,35 @@
 
 'use strict';
 
+// -----------------------------------------------------------------------------
+// Requirements
+// -----------------------------------------------------------------------------
 
-// ------------------------------------------------------------------------------
+/**
+ * @typedef {Object} UploadPart
```

Comment:
In the hopefully near future, the API will pass us back this information directly and we'll pass it directly back to the user, so I think it's useful to retain this typedef over a constructor